tox.ini

```diff
@@ -8,16 +8,16 @@ deps =
     cassandra-driver
     flake8
     isort
+whitelist_externals =
+    bash
 commands =
-    bash -c '                                                                       \
-        python -c "import asyncio, sys; sys.exit(0)" 2>/dev/null                    \
-            && SPECIFIC_TEST_MOD=asyncio || SPECIFIC_TEST_MOD=trollius;             \
-        TESTS_LIST="aiocassandra.py setup.py tests.py tests_$SPECIFIC_TEST_MOD.py"; \
-        flake8 --show-source $TESTS_LIST;                                           \
-        isort --check-only --diff $TESTS_LIST                                       \
-    '
+    bash -c "                                                   \
+        flake8 --show-source aiocassandra.py setup.py tests.py; \
+        isort --check-only aiocassandra.py setup.py tests.py;   \
+    "
 
-    ./tests.py
+    # envpython to mute "WARNING:test command found but not installed in testenv"
+    {envpython} tests.py
```

Comment:
better rely on env python detection of tox. tests should be refactored  anyway

Reply:
mind replacing it w/ pytest?

Reply:
yes, makes perfect sense